packages/paste-core/components/visual-picker/src/VisualPickerCheckboxGroup.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
```

Comment:
(sorry, commenting on a random line because I can't find where spacing is set)

todo: Set space between items to 8px. And space between legend/help text and items to 12px. We decided to match the layout of visual picker to file uploader in crit.